snippets/local.defaults.properties

```diff
@@ -0,0 +1,2 @@
+PLACES_API_KEY="YOUR_API_KEY"
+MAPS_API_KEY="YOUR_API_KEY"
```

Comment:
Since this app doesn't seem to need a Maps API key, can we omit this?

Reply:
Done